samples/boards/nrf/mesh/onoff_level_lighting_vnd_app/src/app_gpio.c

```diff
@@ -64,8 +64,7 @@ void app_gpio_init(void)
 
 	button_device[0] = device_get_binding(DT_GPIO_LABEL(SW0_NODE, gpios));
 	gpio_pin_configure(button_device[0], DT_GPIO_PIN(SW0_NODE, gpios),
-			   GPIO_INPUT | GPIO_INT_DEBOUNCE |
```

Comment:
Removing the debounce flag here seems to change the expected behavior. The sample documents that it works with nrf52840dk_nrf52840, so it seems like this should at least add an overlay for that board specifying the right flags there, no? I have the same question for other similar changes elsewhere in this patch.

Reply:
Or maybe all the relevant board DTS files need to have the debounce flags added to their button nodes?

Reply:
@mbolivar-nordic That's the thing, really. The sample support `nrf52840dk_nrf52840`, but the nRF GPIO driver does not support `GPIO_INT_DEBOUNCE` at all, so this flag has no effect.

One could argue that all button nodes for boards with a SoC supporting GPIO debouncing should have that flag added, but this is no simple task (60+ boards from what I count) - and some of these may be doing hardware button debouncing, rendering this flag unnecessarily or even counterproductive.

As for the other drivers where I have removed this flag, they are drivers for external LoRa chips where the interrupt lines where configured with debouncing enabled. This should not be needed at all.

Reply:
OK, fair enoguh